data/socials.toml

```diff
@@ -0,0 +1,35 @@
+socials = [
```

Comment:
We can't put `socials.toml` into the theme repo. This is equivalent to the default data, and every site that uses this theme renders these.

Reply:
Would it be OK if I put it in ```docs/examples/``` or ```exampleSite```?

Reply:
I've been doing a whole bunch of work on the theme for my site (robot-one.github.io, some domain name problems) and moved the full example socials.toml as well as other stuff to `exampleSite` for the time being. Most in another branch which I will sync with this one after a bit more work.

## Added
- params for round/unset avatar
- increased size of site title
- example images, favicons, css, socials.toml (with lots of URI schemes for almost every use case)
- [WIP] local JS, rather than CDN JS (I still gotta make this conditional - CDN stuff is commented)
- params for moving Archive/Tags/Categories icons to the navbar
- [WIP] adding PWA manifest and service worker; should hopefully help with caching the local JS

## Want to add
- compatibility fixes for Internet Explorer, Firefox, iOS safari (the site breaks in all those browsers and loads the About flip page on top of, or just below, the main article selection)
- performance fixes: scrolling is sometimes juddery, and the navbar doesn't always stick cleanly to the top of the page
- option to use 2 avatars, one for the main homepage, 1 for the About flip page
- separate out the flip page into its own separate thing? since it seems to be part of the main page at the moment and loads on top of it when CSS breaks.
- remove (or make optional) custom scrollbars and jQuery
- add Photoswipe
- maybe add [StPageFlip](https://nodlik.github.io/StPageFlip/)???

Reply:
This list of Added/want to add is slightly outdated. Please see CHANGELOG.md in the latest commits for a full list. No idea what version number to use.